services/net/transcription/Config/TranscriptionOptions.cs

```diff
@@ -0,0 +1,49 @@
+
+using TNO.Services.Config;
+
+namespace TNO.Services.Transcription.Config;
+
+/// <summary>
+/// TranscriptionOptions class, configuration options for transcription service
+/// </summary>
+public class TranscriptionOptions : ServiceOptions
+{
+    #region Properties
+    /// <summary>
+    /// get/set - An array of topics to consume.
```

Comment:
Comma separated list of topics

Reply:
Fixed.

Reply:
This property `Topics` should represent one or more topics the TranscriptionService will listen to.

Reply:
Even for transcription? I thought we had discussed only having one topic for transcription.

Reply:
It will most likely be one, but use the `Topics` property for now.

Reply:
Fixed.